pkg/plugins/plugins.go

```diff
@@ -65,6 +65,7 @@ type PluginDTO struct {
 
 	// temporary
 	backend.StreamHandler
+	backend.CollectMetricsHandler
```

Comment:
this doesn't seem ideal, but it's not clear if there's another option at the moment. @wbrowne or @marefr does this seem ok for now?